File: entry_point/fsharp_entry_point/entry_point.fs
//#light
open System

//comments
(* commments *)

[<EntryPoint>]
let main args =
  let argv = Environment.GetCommandLineArgs()

  if argv.Length <> 2 then
    eprintfn "Usage: %s <arg1>" argv.[0]
    -1
  else
    printfn "code here"
    0

File: entry_point/csharp_entry_point/entry_point.cs
using System;

public class EntryPoint {
  public static void Main( string[] args ) {

    if( args.Length != 1 ) {
      Console.Error.WriteLine( "Usage: " + Environment.GetCommandLineArgs()[0] + " <arg1>" );
      Environment.Exit( 1 );
    }

    Console.WriteLine( args[1] );
  }
}
